package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
d: A second package json? I think the `package json` and the `yarn.lock` on this level can be deleted. It seems that you've initialized the angular project in the `meshstack-hub` folder. This creates `meshstack-hub /meshstack-hub`   hierachy. You need to reference directly the repo folder. So we have only one level.

Reply:
yea fixed